src/server/controllers/Grbl/GrblController.js

```diff
@@ -1952,7 +1952,7 @@ class GrblController {
                     axes.F = axes.F.toFixed(3);
                 }
 
-                const jogCommand = `$J=${unitModal}G91 ` + map(axes, (value, letter) => ('' + letter.toUpperCase() + value)).join(' ');
+                const jogCommand = `$J=${unitModal}G91 ` + map(axes, (value, letter) => ('' + (letter === 'a' || letter === 'A' ? 'Y' : letter.toUpperCase()) + value)).join(' ');
```

Comment:
I don't think you need to change A to Y here. If handled correctly, A-axis movements will be changed to the Y-axis in the front end and there is no need to do a check here for it again.